content/tutorial/tutorial.md

```diff
@@ -709,7 +709,7 @@ function calculateWinner(squares) {
   ];
   for (let i = 0; i < lines.length; i++) {
     const [a, b, c] = lines[i];
-    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+    if (squares[a] === squares[b] && squares[a] === squares[c]) {
```

Comment:
Given
```
null,null,null
X,X,X
O,O,null
```
This function would now return `null` when the winner is `X`

Reply:
Nice catch @eps1lon . This actually change the behavior of calculatewinner function. I had not tested for all win cases so its fault on my part.

Given:
```
null,null,null
X,X,X
O,O,null
```

Explanation:
when checking for indexes [0,1,2]
```
null === null && null === null
this return true
```
so calculatewinner function cannot check for indexes [3,4,5]

```
null && null === null && null === null
this returns null
```
so calculate winner can check for index [3,4,5]

Thanks for the review, I will close this PR